core/garment/src/garment.ts

```diff
@@ -919,7 +919,7 @@ async function garmentFromWorkspace(
 
         if (
           subscription.type === 'glob' &&
-          event.path.startsWith(subscription.input.rootDir)
+          event.path.startsWith(Path.normalize(subscription.input.rootDir))
```

Comment:
@beshanoe I'm not really comfortable with this change.  Though it does work, it adds extra file input-level subscriptions to allCachedSubscriptions.

Before fix:

```
{
    "type": "file",
    "level": "input",
    "path": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\babel.config.js",
    "targetPath": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/accordion/src/context/focus-provider/FocusProvider.js"
}, {
    "type": "glob",
    "level": "input",
    "input": {
        "rootDir": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/typography/src",
        "include": ["**/*.js?(x)"],
        "exclude": ["**/*.test.js", "**/__fixtures__", "**/__mocks__"]
    }
}, {
    "type": "file",
    "level": "input",
    "path": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\babel.config.js",
    "targetPath": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/typography/src/index.js"
}
```

After fix:

```
{
    "type": "file",
    "level": "input",
    "path": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\babel.config.js",
    "targetPath": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/accordion/src/context/focus-provider/FocusProvider.js"
}, {
    "type": "file",
    "level": "input",
    "path": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\babel.config.js",
    "targetPath": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\packages\\accordion\\src\\components\\AccordionButton.js"
}, {
    "type": "glob",
    "level": "input",
    "input": {
        "rootDir": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/typography/src",
        "include": ["**/*.js?(x)"],
        "exclude": ["**/*.test.js", "**/__fixtures__", "**/__mocks__"]
    }
}, {
    "type": "file",
    "level": "input",
    "path": "C:\\Users\\stephan.noel\\farfetch-repos\\ui-box\\babel.config.js",
    "targetPath": "C:/Users/stephan.noel/farfetch-repos/ui-box/packages/typography/src/index.js"
}
```
We could try the approach of changing what's stored in the cache itself, but this would include changing the createdInput, rootDir (probably in getActionGraph) and targetPath to be OS-specific.  Not sure if the TypeScript compiler approach of dealing with only one kind of path internally is a good strategy?

![image (27)](https://user-images.githubusercontent.com/34415120/99861972-0854eb00-2b90-11eb-85ac-044fcf813e30.png)

TLDR;

This "fix" should work for now, but it has some unwanted effects.  Taking the approach of platform-specific paths in the cache itself may imply changes in a lot of places.  Which do you think is preferable?

Reply:
Changed the `changesBySubscription` entry to normalize as suggested, but using `normalizePath`.  It's now working without extra entries in `allSubscriptionsCached`.

It is failing in build due to an outdated github action.  I can try looking into it and after it should be ready to merge.

Reply:
can we use `normalizePath` on this line too? Just to make it look unified. Or they work differently in this case?

Reply:
Sure thing.  They basically do the opposite, so I can just use `normalize-path` on `event.path` instead of using `Path.normalize` on `subscription.input.rootDir`.  What do you think?

Reply:
Looks good to me